guide/getting-started/hardware.md

```diff
@@ -77,7 +79,7 @@ You may already be familiar with physical security keys from your bank or work.
    height = 400
 %}
 
-Bitcoin hardware wallets, also known as external signers, act like bitcoin-centric security keys. They isolate the recovery phrase and private keys from the internet and other devices.
+Bitcoin hardware wallets act like bitcoin-centric security keys. They isolate the recovery phrase, private keys and other sensitive data from the internet and other devices.
```

Comment:
What's an example of "other sensitive data" for a hardware wallet?

Reply:
Descriptors / xpub are two that come to mind.

Reply:
xpub is derived from the PK so it isn't really stored, descriptors though are stored though - I'm pretty sure this is the case with Coldcard (one of the only HWW fully supporting descriptors atm).